project/app/api/update.py

```diff
@@ -56,21 +56,22 @@ def lowerify(text):
 
 load_dotenv()
 
+# globalize these variables because I need to
```

Comment:
code comments ought to describe 'why' or 'what' specifically about the code is important. 

"because I need to" isn't very descriptive as to 'why' they needed to be be globalized.